src/events/GuildBanAddEvent.ts

```diff
@@ -1,12 +1,27 @@
 import BaseEvent from "../structures/BaseEvent";
 import DiscordClient from "../client/client";
-import { GuildBan } from "discord.js";
+import { GuildBan, MessageEmbed } from "discord.js";
+import { getAudituser, ModLog } from "../utils/functions/mod";
 
 export default class Event extends BaseEvent {
     constructor() {
         super("guildBanAdd");
     }
     async run(client: DiscordClient,ban:GuildBan) {
-        
+        var data = await getAudituser(ban)
+        var reason;
+        if(ban.reason){
```

Comment:
This could be done in a better way 
`
const reason = ban.reason ? ban.reason : "No Reason Provided"
`